.github/workflows/go.yml

```diff
@@ -154,7 +154,5 @@ jobs:
       - name: Check out code into the Go module directory
         uses: actions/checkout@v2
 
-      - name: Run golangci-lint 1.30.0
+      - name: Run golangci-lint
         uses: actions-contrib/golangci-lint@v1
-        with:
-          golangci_lint_version: 1.30.0
```

Comment:
So, without passing in the golangci_lint_version, it'll default to v1.23.0. Is that the intended behaviour? 

https://github.com/actions-contrib/golangci-lint/blob/master/Dockerfile

Reply:
Removed from the right file instead :)